ruby-challenges.rb

```diff
@@ -19,13 +27,43 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
-
+def sum_array(array)
+  puts array.sum
+end
+puts sum_array(nums_array1)
+puts sum_array(nums_array2)
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
-
+class Bike
+  attr_accessor :model, :wheels, :current_speed
+  def initialize (model, wheels, current_speed)
```

Comment:
Since you are providing default values to the wheels and speed, you can make your life a little easier by not requiring them on initialization.